ingestclient/core/engine.py

```diff
@@ -341,9 +340,13 @@ def run(self):
                                                  key_parts["t_index"])
 
             try:
+                print("x_tile_size: " + str(self.config.config_data['ingest_job']["tile_size"]["x"]))
+                print("x_tile_size: " + str(self.config.config_data['ingest_job']["tile_size"]["x"]))
```

Comment:
You're print x_tile_size twice.

Reply:
Thanks. I removed them.  They were there to make sure I was getting data.